pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -175,6 +175,13 @@ def test_corr_int(self):
         df3.cov()
         df3.corr()
 
+    @td.skip_if_no_scipy
+    def test_corr_axes(self):
+        # https://github.com/pandas-dev/pandas/issues/35002
+        df = pd.DataFrame(np.random.normal(size=(10, 2)))
+        for meth in ["pearson", "kendall", "spearman"]:
+            tm.assert_frame_equal(df.T.corr(meth, axis=1), df.corr(meth, axis=0))
```

Comment:
Use `result = df.corr(meth, axis=0)`, `expected = df.T.corr(meth, axis=1)` and then assert the equality of both (minor nit but the first case should probably be `df.corr(methd, axis=1)` and the second `axis=0` for clarity since we're testing the non-default value, but obviously it's equivalent)